### PR TITLE
🦄 refactor: Migrated `TranslationBannerLegal.tsx` to shadcn

### PR DESCRIPTION
**Pull request by ameeetgaikwad**
## Related Issue
#13946

### CODE REVIEW

**Comment by Hliwa1990** [comment]
> 
> 
> 
> 
> ## Description
> 
> 
> 
> 
> 
> 
> 
> ## Related Issue
> 
> #13946 
> 
> 
> 
> 
> 
> 
> 
> 
> 
>

File: src/components/TranslationBannerLegal.tsx [hunk 1]
```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
...
-            flexDirection={{ base: "column-reverse", sm: "row" }}
-            mb={4}
-          >
-            <Heading
-              as="h3"
-              fontSize={{ base: "1.25rem", md: "1.5rem" }}
-              fontWeight="bold"
-              lineHeight="100%"
-            >
+        <div className="m-4 mt-[40px] flex flex-col sm:mt-4">
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:53]:
```suggestion
        <div className="m-4 mt-10 flex flex-col sm:mt-4">
```

File: src/components/TranslationBannerLegal.tsx [hunk 2]
```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
...
             <Translation id="translation-banner-no-bugs-content" />
-          </Text>
-          <Flex
-            align={{ base: "flex-start", sm: "center" }}
-            flexDirection={{ base: "column", sm: "row" }}
-          >
+          </p>
+          <div className="flex flex-col items-start sm:flex-row sm:items-center">
             <Button
+              variant="link"
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:68]:
```suggestion
```

File: src/components/TranslationBannerLegal.tsx [hunk 3]
```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
...
-            mb={4}
-          >
-            <Heading
-              as="h3"
-              fontSize={{ base: "1.25rem", md: "1.5rem" }}
-              fontWeight="bold"
-              lineHeight="100%"
-            >
+        <div className="m-4 mt-[40px] flex flex-col sm:mt-4">
+          <div className="mb-4 flex flex-col-reverse items-start sm:flex-row sm:items-center">
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:54]: lets use a gap in the parent to simplify and make the spacing more consistent
```suggestion
          <div className="flex flex-col-reverse items-start sm:flex-row sm:items-center">
```

File: src/components/TranslationBannerLegal.tsx [hunk 4]
```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
...
-        boxShadow={{
-          base: `-4px 10px 0px ${text} 10%`,
-          md: "rgba(0, 0, 0, 0.16) 0px 2px 4px 0px",
-        }}
+      <div
+        className={cn(
+          "relative flex justify-between",
+          "w-full md:max-w-[600px]",
+          "rounded-sm p-4",
+          "bg-info-banner text-black-300",
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:49]:
```suggestion
          "bg-background-highlight text-black-300",
```

File: src/components/TranslationBannerLegal.tsx [hunk 5]
```diff
@@ -96,21 +76,21 @@ const TranslationBannerLegal = ({
...
-          top={0}
-          insetInlineEnd="0"
-          margin={4}
-          color="secondary"
-          _hover={{
-            color: "primary.base",
-          }}
+          </div>
+        </div>
+        <button
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:81]: Lets use our Button component to get the same focus styles

File: src/components/TranslationBannerLegal.tsx [hunk 6]
```diff
@@ -38,54 +33,39 @@ const TranslationBannerLegal = ({
...
-    <Box
-      as="aside"
-      position="fixed"
-      display={isOpen ? "block" : "none"}
-      bottom={{ base: 0, md: 8 }}
-      insetInlineEnd={{ base: 0, md: 8 }}
-      zIndex="99"
+    <aside
+      className={cn(
+        "fixed z-50",
```
Comment (pettinarip) [inline src/components/TranslationBannerLegal.tsx:38]:
```suggestion
        "fixed z-popover",
```

**Review by pettinarip — COMMENTED** [review]
Thanks for the PR @ameeetgaikwad, looks good. Im pushing a few minor changes to simplify the styles and make it more consistent with the rest of the migration.

**Review by pettinarip — APPROVED** [review]

**Pull request merged**